testcases/misc_tests/cca_export_import_test.c

```diff
@@ -748,7 +798,47 @@ static CK_RV cca_aes_data_export_import_tests(void)
             goto error;
         }
 
-        testcase_pass("CCA export/import test with AES%u data key: ok", 8 * keylen);
+        /* Set extractable to FALSE on original key */
+        rc = extract_restrict(session, hkey);
+        if (rc != CKR_OK) {
+            testcase_fail("extract_restrict rc=%s", p11_get_ckr(rc));
+            goto error;
+        }
+
+        /* Convert to CIPHER key */
+        rc = convert_to_cipher_key(session, hkey);
+        if (rc != CKR_OK) {
+            testcase_fail("extract_restrict rc=%s", p11_get_ckr(rc));
+            goto error;
+        }
+
+        /* encrypt some data with this converted key */
+
+        rc = funcs->C_EncryptInit(session, &mech, hkey);
+        if (rc != CKR_OK) {
+            testcase_error("C_EncryptInit rc=%s", p11_get_ckr(rc));
+            goto error;
+        }
+        ilen = len = sizeof(data);
+        rc = funcs->C_Encrypt(session, data, ilen, encdata3, &len);
+        if (rc != CKR_OK) {
+            testcase_error("C_Encrypt rc=%s", p11_get_ckr(rc));
+            goto error;
+        }
+        if (ilen != len) {
+            testcase_fail("plain and encrypted data len does not match");
```

Comment:
do not match

Reply:
hm, well the 'does' is for the 'len', not for the 'plain and encrypted data', so IMHO 'does' is correct.